docs/actions.md

```diff
@@ -887,9 +898,7 @@ You can use the OpenWhisk CLI to watch the output of actions as they are invoked
   ```
   wsk activation poll
   ```
-
-  This command starts a polling loop that continuously checks for logs from activations.
```

Comment:
Looks like it got deleted when I was removed polling + --last doc stuff, will add back.